### ISSUE TITLE
An error while Style/TrailingCommaInArguments cop on lamda.(call)

### ISSUE THREAD

**Issue by Tietew** [rubocop-hq/rubocop#7532]
Rubocop reports an error ```undefined method `line' for nil:NilClass``` when:
- Style/TrailingCommaInArguments.EnforcedStyleForMultiline is `consistent_comma`
- lambda is called in style `func.()`
- arguments are multiline

Both `func.call(args)` and `func[args]` styles are passed normally.
If lambda call is single line, passed normally.
If last comma is omitted, _same error occurs_ and no offenses detected unexpectedly.
If EnforcedStyleForMultiline is `comma` or `no_comma`, passed normally.

## Expected behavior

No errors occur.

## Actual behavior

Rubocop reports 1 error:

```
For ...: configuration from .../.rubocop.yml
Default configuration from .../gems/rubocop-0.77.0/config/default.yml
Inspecting 1 file
Scanning .../test.rb
An error occurred while Style/TrailingCommaInArguments cop was inspecting .../test.rb:4:0.
undefined method `line' for nil:NilClass
.../gems/rubocop-0.77.0/lib/rubocop/cop/mixin/trailing_comma.rb:97:in `method_name_and_arguments_on_same_line?'
.../gems/rubocop-0.77.0/lib/rubocop/cop/mixin/trailing_comma.rb:70:in `should_have_comma?'
.../gems/rubocop-0.77.0/lib/rubocop/cop/mixin/trailing_comma.rb:38:in `check_comma'
.../gems/rubocop-0.77.0/lib/rubocop/cop/mixin/trailing_comma.rb:24:in `check'
.../gems/rubocop-0.77.0/lib/rubocop/cop/style/trailing_comma_in_arguments.rb:58:in `on_send'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:136:in `with_cop_error_handling'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:56:in `block in trigger_responding_cops'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:55:in `each'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:55:in `trigger_responding_cops'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:32:in `block (2 levels) in <class:Commissioner>'
.../gems/rubocop-0.77.0/lib/rubocop/ast/traversal.rb:50:in `block in on_begin'
.../gems/rubocop-0.77.0/lib/rubocop/ast/traversal.rb:50:in `each'
.../gems/rubocop-0.77.0/lib/rubocop/ast/traversal.rb:50:in `on_begin'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
.../gems/rubocop-0.77.0/lib/rubocop/ast/traversal.rb:13:in `walk'
.../gems/rubocop-0.77.0/lib/rubocop/cop/commissioner.rb:44:in `investigate'
.../gems/rubocop-0.77.0/lib/rubocop/cop/team.rb:124:in `investigate'
.../gems/rubocop-0.77.0/lib/rubocop/cop/team.rb:112:in `offenses'
.../gems/rubocop-0.77.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:296:in `inspect_file'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:244:in `block in do_inspection_loop'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:276:in `block in iterate_until_no_changes'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:269:in `loop'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:269:in `iterate_until_no_changes'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:240:in `do_inspection_loop'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:119:in `block in file_offenses'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:143:in `file_offense_cache'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:117:in `file_offenses'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:108:in `process_file'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:87:in `block in each_inspected_file'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:86:in `each'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:86:in `reduce'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:86:in `each_inspected_file'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:73:in `inspect_files'
.../gems/rubocop-0.77.0/lib/rubocop/runner.rb:39:in `run'
.../gems/rubocop-0.77.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
.../gems/rubocop-0.77.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
.../gems/rubocop-0.77.0/lib/rubocop/cli/command.rb:10:in `run'
.../gems/rubocop-0.77.0/lib/rubocop/cli/environment.rb:17:in `run'
.../gems/rubocop-0.77.0/lib/rubocop/cli.rb:65:in `run_command'
.../gems/rubocop-0.77.0/lib/rubocop/cli.rb:72:in `execute_runners'
.../gems/rubocop-0.77.0/lib/rubocop/cli.rb:41:in `run'
.../gems/rubocop-0.77.0/exe/rubocop:13:in `block in <top (required)>'
.../lib/ruby/2.6.0/benchmark.rb:308:in `realtime'
.../gems/rubocop-0.77.0/exe/rubocop:12:in `<top (required)>'
.../bin/rubocop:23:in `load'
.../bin/rubocop:23:in `<top (required)>'
.../gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `load'
.../gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
.../gems/bundler-2.0.2/lib/bundler/cli/exec.rb:28:in `run'
.../gems/bundler-2.0.2/lib/bundler/cli.rb:465:in `exec'
.../gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
.../gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
.../gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
.../gems/bundler-2.0.2/lib/bundler/cli.rb:27:in `dispatch'
.../gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
.../gems/bundler-2.0.2/lib/bundler/cli.rb:18:in `start'
.../gems/bundler-2.0.2/exe/bundle:30:in `block in <top (required)>'
.../gems/bundler-2.0.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
.../gems/bundler-2.0.2/exe/bundle:22:in `<top (required)>'
.../bin/bundle:23:in `load'
.../bin/bundle:23:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Style/TrailingCommaInArguments cop was inspecting .../test.rb:4:0.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.77.0 (using Parser 2.6.5.0, running on ruby 2.6.5 x86_64-linux)
Finished in 0.11265576106961817 seconds
```

### Gemfile
``` ruby
source "https://rubygems.org"
gem "rubocop"
```

### .rubocop.yml
``` yaml
AllCops:
  TargetRubyVersion: "2.6.0"
Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: consistent_comma
Style/LambdaCall:
  Enabled: false
```

### test.rb
``` ruby
# frozen_string_literal: true

func = ->(_a, _b, _c) { something }
func.(
  'foo',
  'bar',
  'baz',
)
```

### run
```
$ bundle exec rubocop test.rb -d
```

## RuboCop version

```
0.77.0 (using Parser 2.6.5.0, running on ruby 2.6.5 x86_64-linux)
```

**Comment by Tietew** [comment]
👍
